test/docs/check.py

```
@@ -0,0 +1,657 @@
+    file_type = detect_file_type(path, lines)
+
+    if file_type == "case":
+        return validate_case(path, lines)
+    elif file_type == "plan":
+        return validate_plan(path, lines)
+    elif file_type == "matrix":
+        return validate_matrix(path, lines)
+    else:
+        return [ValidationError(
+            relative_path(path), None,
+            f"could not detect file type (expected case, plan, or matrix)",
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)